node/src/Transaction.ts

```diff
@@ -3,7 +3,8 @@
  */
 
 import {
-    BaseClient, // eslint-disable-line @typescript-eslint/no-unused-vars
+    BaseClient,
+    GlideRecord, // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
move eslint back otherwise it will blame you

Reply:
oh no!